src/cordova/apple/OutlineTun2Socks/Package.resolved

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Libraries shouldn't have resolved files. That's really for things we build and publish.

The consumers of this package may want to pick other versions.